docs/tutorials/rust.md

```diff
@@ -0,0 +1,152 @@
+# Tutorial: Develop a Rust package using `pixi`
+
+In this tutorial, we will show you how to develop a Rust package using `pixi`.
+The tutorial is written to be executed from top to bottom, missing steps might result in errors.
+
+The audience for this tutorial is developers who are familiar with Rust and `cargo` and how are interested to try pixi for their development workflow.
+
+!!! note ""
+    If you're new to pixi, you can check out the [basic usage](../basic_usage.md) guide.
+    This will teach you the basics of pixi project within 3 minutes.
+
+## Prerequisites
+
+- You need to have `pixi` installed. If you haven't installed it yet, you can follow the instructions in the [installation guide](../index.md).
+  The crux of this tutorial is to show you only need pixi!
+
+## Create a pixi project.
+
+```shell
+pixi init my_rust_project
+cd my_rust_project
+```
+
+It should have created a directory structure like this:
+
+```shell
+my_rust_project
+├── .gitattributes
+├── .gitignore
+└── pixi.toml
+```
+
+The `pixi.toml` file is the manifest file for your project. It should look like this:
+
+```toml title="pixi.toml"
+[project]
+name = "my_rust_project"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["User Name <user.name@email.url>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tasks]
+
+[dependencies]
+```
+
+## Add Rust dependencies
+
+To use a pixi project you don't need any dependencies on your system, all the dependencies you need should be added through pixi, so other users can use your project without any issues.
+```shell
+pixi add rust
+```
+
+This will add the `rust` package to your `pixi.toml` file.
+
+## Add a `cargo` project
+As you now have rust installed, you can create a `cargo` project in your `pixi` project.
+```shell
+pixi run cargo init
```

Comment:
I would add here maybe as a note which will briefly explain how `pixi run` works so it would be clear for a novice user why we write
pixi run cargo init